screed/tests/test_convert.py

```diff
@@ -13,10 +15,18 @@ class Test_fasta_to_fastq(test_fasta.Test_fasta):
     """
 
     def setup(self):
-        thisdir = os.path.dirname(__file__)
-        self._fqName = os.path.join(thisdir, 'fa_to_fq')
-        self._faName = os.path.join(thisdir, 'fq_to_fa')
-        self._testfa = os.path.join(thisdir, 'test.fa')
+
+        temp_fa2fq = utils.get_temp_filename('fa_to_fq')
+        # shutil.copy(os.path.join(thisdir, 'fa_to_fq'), temp_fa2fq)
```

Comment:
Can you get this new section of code down to three lines like the original? No need to leave in commented out coffee and one use variables.